packages/main/test/specs/Toast.spec.js

```diff
@@ -0,0 +1,116 @@
+		const button = browser.$("#wcBtnShowToastBC");
+		const toast = browser.$("#wcToastBC");
+
+		button.click();
+
+		assert.strictEqual(toast.getProperty("placement"), "BottomCenter",
+			"Placement should be the default BottomCenter");
+	});
+
+	it("tests shadow content div role", () => {
+		const toastShadowContent = browser.$("#wcToastBE").shadow$(".ui5-toast-content");
+
+		assert.strictEqual(toastShadowContent.getAttribute("role"), "alert",
+			"The role of the shadow ui5-toast-content div should be alert");
+	});
+
+	it("tests shadow content div inline styles with default duration", () => {
+		const button = browser.$("#wcBtnShowToastBE");
+		const toastShadowContent = browser.$("#wcToastBE").shadow$(".ui5-toast-content");
+
+		button.click();
+
+		assert.strictEqual(toastShadowContent.getAttribute("style"),
+			"transition-duration: 1000ms; transition-delay: 2000ms; opacity: 0;",
+			"The correct default inline styles are applied to the shadow ui5-toast-content");
+	});
+
+	it("tests shadow content div inline styles with long duration", () => {
+		const button = browser.$("#wcBtnShowToastBS");
+		const toast = browser.$("#wcToastBS");
+		const toastShadowContent = toast.shadow$(".ui5-toast-content");
+		const maximumAllowedTransition = 1000;
+		const durationProperty = toast.getProperty("duration");
+		let calculatedDelay;
+
+		button.click();
+
+		calculatedDelay = `${durationProperty - maximumAllowedTransition}ms`;
+
+		assert.strictEqual(toastShadowContent.getAttribute("style"),
+				`transition-duration: ${maximumAllowedTransition}ms; transition-delay: ${calculatedDelay}; opacity: 0;`,
+				"The correct custom inline styles are applied to the shadow ui5-toast-content," +
+				"when the duration is longer than default. Transition is not longer than allowed (1000ms).");
+	});
+
+	it("tests shadow content div inline styles with short duration", () => {
+		const button = browser.$("#wcBtnShowToastME");
+		const toast = browser.$("#wcToastME");
+		const toastShadowContent = toast.shadow$(".ui5-toast-content");
+		const durationProperty = toast.getProperty("duration");
+		let calculatedTransition, calculatedDelay;
+
+		button.click();
+
+		calculatedTransition = durationProperty / 3;
+		calculatedDelay = `${durationProperty - calculatedTransition}ms`;
+
+		assert.strictEqual(toastShadowContent.getAttribute("style"),
+				`transition-duration: ${calculatedTransition}ms; transition-delay: ${calculatedDelay}; opacity: 0;`,
+				"The correct custom inline styles are applied to the shadow ui5-toast-content," +
+				"when the duration is shorter than default. Transition is a third of the duration.");
+	});
+
+	it("tests closing of toast", () => {
+		const button = browser.$("#wcBtnShowToastMS");
+		const toast = browser.$("#wcToastMS");
+		const toastShadowContent = toast.shadow$(".ui5-toast-content");
+
+		button.click();
+
+		// Give time for the animation to end and _ontransitionend to be called
+		browser.pause(3500);
```

Comment:
Do the test with a toast with short duration, so the pause can be shorter and the test faster.